### DISCUSSION TITLE
Contribution reg.

### DISCUSSION THREAD

**Discussion by nivramam** [python-telegram-bot/python-telegram-bot#2642] [Q&A]
Hello there, this is Niveydhithaa, final year student at PSG College of Technology (Coimbatore, India). As part of my course "Open Source Software Lab", I would like to understand and contribute to this python-telegram-bot repo as I am interested in this domain. I have joined the Telegram group and found it very useful for starting on. 

I just had a clarification as to whether is there any procedure to begin my contributions journey and do I need to send an e-mail to the admins? Thanks a lot in advance, and waiting to get started!

**Comment by Bibo-Joshi** [comment]
Hi. Glad you are enjoying PTB :)

We have no standard protocol for new contributers. You should definetely give the [contribution guide](https://github.com/python-telegram-bot/python-telegram-bot/blob/v14/.github/CONTRIBUTING.rst) a read. Note that we're currently working towards a majorly breaking release, so branches & pull requests should be based on the `v14` branch.
Do you have anything in mind already that you'd like to do?

**Reply by nivramam** [reply]
Oh that's great! Thanks a lot, sure will go through the guidelines, and currently I am not yet ready with any issue, just trying to get started. Thanks again
